resources/assets/js/validations.js

```diff
@@ -410,100 +380,68 @@ $.extend(true, laravelValidation, {
                 }
             }
             var regex = new RegExp("^(?:"+matches[1]+")$",php_modifiers.join());
-            return  this.optional(element) || regex.test(value);
-        }, $.validator.format("The :attribute format is invalid."));
+            return   regex.test(value);
+        },
 
         /**
          * Validate that an attribute is a valid date.
+         * @return {boolean}
          */
-        $.validator.addMethod("laravelDate", function(value, element, params) {
-            return this.optional(element) ||(helpers.strtotime(value)!=false);
-        }, $.validator.format("The :attribute is not a valid date"));
+        Date: function(value) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620834/issues/source?bid=2225045&fileBranchId=2805859#l391)